nautobot/extras/datasources/git.py

```diff
@@ -535,6 +535,12 @@ def import_config_context(context_data, repository_record, job_result, logger):
     context_metadata.setdefault("description", "")
     context_metadata.setdefault("is_active", True)
 
+    # Context Metadata `schema` has been updated to `config_context_schema`,
+    # but for backwards compatibility `schema` is still supported.
+    if "schema" in context_metadata and "config_context_schema" not in context_metadata:
+        logger.warning("`schema` is deprecated in `_metadata`, please use `config_context_schema` instead.")
```

Comment:
Possibly `job_result.log()` instead of `logger.warning()` so that it shows up in the job logs and the UI?

Reply:
Still a recommendation.